core/api/core.klib.api

```diff
@@ -570,6 +568,8 @@ final val com.revenuecat.purchases.kmp.models/isBasePlan // com.revenuecat.purch
     final fun (com.revenuecat.purchases.kmp.models/SubscriptionOption).<get-isBasePlan>(): kotlin/Boolean // com.revenuecat.purchases.kmp.models/isBasePlan.<get-isBasePlan>|<get-isBasePlan>@com.revenuecat.purchases.kmp.models.SubscriptionOption(){}[0]
 final val com.revenuecat.purchases.kmp.models/isPrepaid // com.revenuecat.purchases.kmp.models/isPrepaid|@com.revenuecat.purchases.kmp.models.SubscriptionOption{}isPrepaid[0]
     final fun (com.revenuecat.purchases.kmp.models/SubscriptionOption).<get-isPrepaid>(): kotlin/Boolean // com.revenuecat.purchases.kmp.models/isPrepaid.<get-isPrepaid>|<get-isPrepaid>@com.revenuecat.purchases.kmp.models.SubscriptionOption(){}[0]
+final val com.revenuecat.purchases.kmp.models/localizedDescription // com.revenuecat.purchases.kmp.models/localizedDescription|@cocoapods.PurchasesHybridCommon.RCStoreProduct{}localizedDescription[0]
+    final fun (cocoapods.PurchasesHybridCommon/RCStoreProduct).<get-localizedDescription>(): kotlin/String? // com.revenuecat.purchases.kmp.models/localizedDescription.<get-localizedDescription>|<get-localizedDescription>@cocoapods.PurchasesHybridCommon.RCStoreProduct(){}[0]
```

Comment:
I was just noticing this but are we exposing the PHC API as well? We should try to avoid that... I see we are using `api` in android: https://github.com/RevenueCat/purchases-kmp/blob/main/core/build.gradle.kts#L15. Is this needed? As for iOS, I'm not sure if there is a way to avoid leaking the API... (This is not a blocker for this PR, but I think we should address before initial release)